draft-packages/menu/KaizenDraft/Menu/README.mdx

```diff
@@ -0,0 +1,63 @@
+---
+title: Menu
+navTitle: Menu
+summaryParagraph: A menu contains links to places or button actions. It does NOT show a selected item at all and the menu button text never changes.
```

Comment:
Nice - the most important information is super clear and prominent!

Reply:
The only kind of exception/edge based on today's discussion: "menu button text never changes". If the page is refreshed, and then the button text changes, did the button text actually change, or is it just a different page with a different button 🤔. This is relevant to whether or not we did actually implement the "heading 1 title block menu" as a Menu instead of a Select, or is this just an edge case where we're willing to break the rule for expediency? We can continue to discuss this outside this PR, but thought it's worth mentioning.